### PR TITLE
fix #62649 soft98.ir

### LINKED ISSUE #62649: soft98.ir
### Issue URL (Ads)
 [https://soft98.ir/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsoft98.ir%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.12
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese, <br/>ABPindo, <br/>EasyList China, <br/>EasyList Czech and Slovak, <br/>EasyList Italy, <br/>Liste AR, <br/>Official Polish filters for AdBlock, uBlock Origin & AdGuard

### PR DESCRIPTION
**Pull request by stanislav-atr**
#62649

Is there a way to block by `a7d2x-` like regex into cosmetic rule or smth?

image

On L6403 and L6404 wich one should we use?

### CODE REVIEW

**Comment by slavaleleka** [comment]
you can select that element by such rule — `div[id$="-special"]`
https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors

**Comment by stanislav-atr** [comment]
> 
> 
> you can select that element by such rule — `##div[id$="-special"]`
> https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors

There are also `#a7d2x-header`, won't that be too broad: `div[id$="-special"]` and `div[id$="-header"]`?

**Comment by BlazDT** [comment]
Please note the site have had adblock detections in the past when the -special elements were hidden.

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -6393,10 +6393,15 @@ soft98.ir#$#[id^="sidebar"][id*="stick"] > section { opacity: 0!important; point
 soft98.ir#$#.a4d2x { display: none!important; }
 soft98.ir#$#.d-js2-none { display: none!important; }
 soft98.ir#$#.d-js2-block { display: block!important; }
+soft98.ir##body > .toast
+soft98.ir##body > div[id][class] > div[class] > section.card
+soft98.ir###content-bottom__inner > aside#sidebar > section.bg-light
+soft98.ir##div[class^="col-12"] > div[id][class] > a > img
```
Comment (zloyden) [inline EnglishFilter/sections/foreign.txt:6399]: `soft98.ir##body > .toast` - I don't see this.
`soft98.ir##body > div[id][class] > div[class] > section.card` - try, for example, `soft98.ir##body > div#content-top + div.container`
`soft98.ir##div[class^="col-12"] > div[id][class] > a > img` - I guess you block this image via URL, so you can remove it

File: EnglishFilter/sections/foreign.txt [hunk 2]
```diff
@@ -6393,10 +6393,15 @@ soft98.ir#$#[id^="sidebar"][id*="stick"] > section { opacity: 0!important; point
...
 !#if (adguard_app_windows || adguard_app_mac || adguard_app_android)
 ! soft98.ir/templates/soft98_default/js/jquery$replace=/\|antibot\|(\|)?[\s\S]*\|main\|/\|antibot\|\$1body\|main\|/
 ||soft98.ir/templates/soft98_default/js/jquery$replace=/\|antibot\|(size\|)?(\|)?[\s\S]*?\|main\|/\|antibot\|\$1\$2body\|main\|/
 !||*.cf^$domain=soft98.ir,image
+||cloudirs.cf^$domain=soft98.ir
```
Comment (zloyden) [inline EnglishFilter/sections/foreign.txt:6404]: If you want to block only images from this domain, add a modificator.

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (stanislav-atr) [reply EnglishFilter/sections/foreign.txt:6399]: .toast is a popup that appears sometimes
image
"To view the link, disable the ad removal plugin in your browser"
Moved to annoyances.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**